### PR TITLE
Adds Elm-specific front-end-homework

### PR DESCRIPTION
**Pull request by flukeout**

### CODE REVIEW

File: front-end-dep-tree/EVALUATION.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## Review Questions
+
+These are the questions that we should ask when reviewing a homework submission.
+
+* Is the app in one of the languages listed in the instructions? If not, did the applicant get special permission to use something else?
+* Does the submittions include instructions for getting the assgignment up and running locally.
+* Does the app meet each of the user story requirements set in the assignment
+* Is the code well organized?
+  * Is it split into multiple methods and views in an appropriate way?
+  * Is there a clear split between the command line program and libraries?
```
Comment (awoitte) [inline front-end-dep-tree/EVALUATION.md:10]: It's no longer a command line app. A program/library split probably isn't that relevant anymore either

File: front-end-dep-tree/README.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+* `xhtml2pdf`
+
+#### Requirements
+
+* The app allows the user to select from the three packages above.
+* By default, no pacakge is selected.
+* When one of the packages is selected it should display information for just that package.
+* When displaying a package it should list the following information...
+  * The package name
+  * A list available versions
```
Comment (awoitte) [inline front-end-dep-tree/README.md:73]: missing an "of" here I think

File: front-end-dep-tree/EVALUATION.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+## Review Questions
+
+These are the questions that we should ask when reviewing a homework submission.
+
+* Is the app in one of the languages listed in the instructions? If not, did the applicant get special permission to use something else?
+* Does the submittions include instructions for getting the assgignment up and running locally.
```
Comment (th3coop) [inline front-end-dep-tree/EVALUATION.md:6]:
```suggestion
* Does the submission include instructions for getting the assignment up and running locally.
```

**Review by awoitte — CHANGES_REQUESTED** [review]

File: front-end-dep-tree/EVALUATION.md (hunk 1 above)
Comment (th3coop) [inline front-end-dep-tree/EVALUATION.md:10]:
```suggestion
  * Is there a clear split between implementation and third party libraries?
```

File: front-end-dep-tree/EVALUATION.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+* Does the app meet each of the user story requirements set in the assignment
+* Is the code well organized?
+  * Is it split into multiple methods and views in an appropriate way?
+  * Is there a clear split between the command line program and libraries?
+  * Does the code include comments when necessary to explain what's going on.
```
Comment (th3coop) [inline front-end-dep-tree/EVALUATION.md:11]:
```suggestion
  * Does the code include comments when necessary to explain what's going on?
```

File: front-end-dep-tree/README.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+that you may not have time to fully polish your work, so focus on code before
+documentation and other niceties.
+
+We strongly encourage the use of third party packages as you work on this
+assignment. However, see the note at the end about providing us with
+instructions on installing/running your assignment.
+
+We are happy to accept solutions provided in the following languages:
+
+* Elm
```
Comment (th3coop) [inline front-end-dep-tree/README.md:18]: I'd change this to not imply there were other alternative.  Just say "We would prefer the assignment is done in Elm."

Then there is the note about doing it in a different language below as long as they ask permission.

**Review by th3coop — CHANGES_REQUESTED** [review]
I'd rename the folder for the assignment. Maybe like "mini-pypi-front-end"

File: front-end-dep-tree/EVALUATION.md (hunk 3 above)
Comment (flukeout) [reply front-end-dep-tree/EVALUATION.md]: Fixed

File: front-end-dep-tree/README.md (hunk 2 above)
Comment (flukeout) [reply front-end-dep-tree/README.md]: Fixed

File: front-end-dep-tree/EVALUATION.md (hunk 1 above)
Comment (flukeout) [reply front-end-dep-tree/EVALUATION.md]: Fixed

Comment (flukeout) [reply front-end-dep-tree/EVALUATION.md]: Fixed

File: front-end-dep-tree/EVALUATION.md (hunk 4 above)
Comment (flukeout) [reply front-end-dep-tree/EVALUATION.md]: Fixed

File: front-end-dep-tree/README.md (hunk 5 above)
Comment (flukeout) [reply front-end-dep-tree/README.md]: Fixed

**Review by awoitte — APPROVED** [review]
Looks great!

File: elm-package-info-viewer/EVALUATION.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  * Does the code include comments when necessary to explain what's going on?
+* Does the code avoid repeating similar blocks of code for similar tasks?
+* Does the code use third party libraries and tools as appropriate?
+
+### Bonus Points
+
+These are not expected as a baseline, but their presence is a good thing.
+
+* Does the app handle loading and error states gracefully
+* Does the app handle missing data gracefully
```
Comment (th3coop) [inline elm-package-info-viewer/EVALUATION.md:19]: Add a note about dynamic sizing.  Doesn't need to be stable on all screen sizes but if it doesn't scale at all that's a big red flag.

**Review by th3coop — CHANGES_REQUESTED** [review]

**Pull request merged**